### PR TITLE
Fixes #179 - Adds NetInfo module.

### LINKED ISSUE #179: Add NetInfoModule

### PR DESCRIPTION
**Pull request by rozele**
Currently, we are mimicking the iOS behavior for the NetInfoModule. Eventually, we can add other Windows specific features in the same way Android has.

### CODE REVIEW

File: ReactWindows/ReactNative.Tests/Modules/NetInfo/NetInfoModuleTests.cs [hunk 1]
```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
+using ReactNative.Modules.NetInfo;
+using Windows.Networking.Connectivity;
+using ReactNative.Modules.Core;
+using ReactNative.Bridge;
+using System.Collections.Generic;
```
Comment (rozele) [inline ReactWindows/ReactNative.Tests/Modules/NetInfo/NetInfoModuleTests.cs]: Sort.

File: ReactWindows/ReactNative/Modules/NetInfo/IConnectionProfile.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace ReactNative.Modules.NetInfo
+{
+    public interface IConnectionProfile
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/NetInfo/IConnectionProfile.cs]: Add comments.

File: ReactWindows/ReactNative/Modules/NetInfo/INetworkInformation.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using Windows.Networking.Connectivity;
+
+namespace ReactNative.Modules.NetInfo
+{
+    public interface INetworkInformation
```
Comment (rozele) [inline ReactWindows/ReactNative/Modules/NetInfo/INetworkInformation.cs]: Comments.

**Comment by ebragge** [comment]
LGTM

**Pull request merged**